### DISCUSSION TITLE
Amazon Echo Show 8 as a cheap remote client

### DISCUSSION THREAD

**Discussion by cgx** [navidrome/navidrome#3305] [Q&A] [answered]
Does anyone listen to music with a **Amazon Echo Show 8** using the integrated web player of Navidrome? The Echo Show looks like a potentially interesting cheap all-in-one client.

Thanks!

**Comment by SickProdigy** [comment, marked as answer]
I'm not sure which echo i have, but a lot of things don't work on it. For 1, there is no web browser. Only apps. So someone would have to make an app, or just use echo's bluetooth to play the music from phone is all i can think.

A much better option I'd try is the google one. 

Another big issue for me, plex doesn't work. It's a glorified alarm clock at this point for me.
